virtcontainers/network.go

```diff
@@ -1437,6 +1437,21 @@ func (n *Network) Add(ctx context.Context, config *NetworkConfig, hypervisor hyp
 	return endpoints, nil
 }
 
+func (n *Network) PostAdd(ctx context.Context, ns *NetworkNamespace, hotplug bool) error {
+	if !hotplug && ns.Endpoints != nil {
```

Comment:
Please can you invert this logic to avoid the deep nesting:

```go
if hotplug {
    return nil
}

if ns.Endpoints == nil {
    return nil
}

endpoints := ns.Endpoints

for _, endpoint := range endpoints {
    // ...
}

```

Reply:
Done